src/class/cdc/cdc_host.c

```diff
@@ -36,6 +37,19 @@
 
 #include "cdc_host.h"
 
+#if CFG_TUH_CDC_FTDI
```

Comment:
is there any reason to move it here, otherwise please keep these header in there section

Reply:
yes
```
  #if CFG_TUH_CDC_PL2303
    pl2303_t pl2303;
  #endif
```
is added to `cdch_interface_t`. so `#include "serial/pl2303.h"` need to be before `typedef struct ... cdch_interface_t`.
And so I moved all includes to top of file for uniformity.